src/app/[locale]/(pages)/esci/errore/page.tsx

```diff
@@ -21,11 +31,16 @@ const LogOutKo = (): React.ReactElement => {
               href: ROUTES.LOGOUT_CONFIRM,
               variant: 'contained',
               text: t('error.retry'),
+              onClick: () =>
+                trackEvent(
+                  isL1 ? 'IO_SESSION_EXIT_TRY_AGAIN' : 'IO_PROFILE_SESSION_EXIT_TRY_AGAIN'
+                ),
             }}
             secondButton={{
               variant: 'outlined',
-              href: ROUTES.PROFILE,
-              text: t('common.backtoprofile'),
+              href: isL1 ? ROUTES.LOGIN : ROUTES.PROFILE,
+              text: isL1 ? t('common.close') : t('common.backtoprofile'),
```

Comment:
Is this a previous bug or regression?

Reply:
doing the task, I realized that it was never corrected